spec/controllers/ops_controller/ops_rbac_spec.rb

```diff
@@ -125,6 +125,35 @@
     end
 
     context "#tenant_set_record_vars" do
+      before do
+        Tenant.seed
+        @tenant = Tenant.root_tenant
+        server = FactoryGirl.create(:miq_server, :zone_id => 1)
+        MiqServer.stub(:my_server).and_return(server)
+      end
+
+      it "saves name in record when use_config_attributes is false" do
+        controller.instance_variable_set(:@_params,
+                                         :divisible                 => true,
+                                         :use_config_for_attributes => "on"
+                                        )
+        controller.send(:tenant_set_record_vars, @tenant)
+        @tenant.name.should eq(MiqServer.my_server.get_config("vmdb").config.fetch_path(:server, :company))
```

Comment:
If possible, get all this `my_server.get_config` code out of the controller and tests.

You can probably use 
```ruby
stub_server_configuration(:server => { :company => "Settings Company Name"})
# and
expect(@tenant.name).to eq "Settings Company Name"
```